OpenNefia.Content/UI/Hud/SimpleMessageWindow.cs

```diff
@@ -46,6 +46,10 @@ public void Print(string text, Color? color = null)
             Messages.PushFront(new ColoredString(text, color.Value));
             NeedsRelayout = true;
         }
+        public void Newline()
```

Comment:
If we don't use `SimpleMessageWindow` anymore, maybe we can delete it. It was only a temporary thing until the real message window was implemented anyway.

Reply:
deleted in 9eb449f